source/connect/favorite-connections/import-export-ui/export.txt

```diff
@@ -0,0 +1,119 @@
+.. _compass-export-connections-ui:
+
+==============================================
+Export Favorite Connections on |compass-short|
+==============================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can export favorite connections on |compass|. Colleagues can import your 
+favorite connections, or you can use them in your other workspaces.
+
+About This Task
+~~~~~~~~~~~~~~~
```

Comment:
This renders fine in staging, but usually we will use the dash `-` underline for h1s, and use the tilde `~` for h2s.